docs/current_docs/partials/_dagger_module_init.mdx

```diff
@@ -85,10 +85,70 @@ In this structure:
 - `dagger.json` is the [Dagger module configuration file](../configuration/modules.mdx).
 - `package.json` manages the module dependencies.
 - `src/` is where your Dagger module code goes. It contains sample code to help you get started.
-- `sdk/` contains the TypeScript SDK.
+- `sdk/` contains the bundled TypeScript SDK.
 
 For examples of modules written in Typescript, see [Daggerverse Modules in Typescript](https://daggerverse.dev/search?sdk=typescript).
 
+<a id="ts-configure-sdk-origin"></a>
```

Comment:
I suggest moving this section under https://docs.dagger.io/configuration/modules#typescript as it is related to configuration and if you agree, also updating the link in the `module-structure.mdx` page

Reply:
Totally agree!

Reply:
Updated :)